sdk/migrationdiscoverysap/Azure.ResourceManager.MigrationDiscoverySap/tests/Tests/MigrationSapDiscoveryTests.cs

```diff
@@ -0,0 +1,333 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.ResourceManager.MigrationDiscoverySap.Models;
+using Azure.ResourceManager.MigrationDiscoverySap.Tests.Tests.Enums;
+using Azure.ResourceManager.Models;
+using Azure.ResourceManager.Resources;
+using Newtonsoft.Json.Linq;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.MigrationDiscoverySap.Tests.Tests
+{
+    public class MigrationSapDiscoveryTests : MigrationDiscoverySapManagementTestBase
+    {
+        public MigrationSapDiscoveryTests(bool isAsync) : base(isAsync)
+        {
+        }
+
+        public const string migrateProjectIdFormat = "/subscriptions/{0}/resourceGroups/{1}/providers/Microsoft.Migrate/MigrateProjects/{2}";
+
+        [TestCase]
+        public async Task TestDiscoveryOperations()
+        {
+            SubscriptionResource subscription = await Client.GetDefaultSubscriptionAsync();
```

Comment:
why move it from createCommonClient method to here?

Reply:
Done